### ISSUE TITLE
Using Unsloth in conjunction with GRPO to train a model for OOM

### ISSUE THREAD

**Issue by chenhao324** [modelscope/ms-swift#3183]
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7  
NPROC_PER_NODE=7  
swift rlhf  
--rlhf_type grpo  
--model /home/huangjun/.cache/modelscope/hub/qwen/Qwen2-VL-7B-Instruct  
--reward_funcs accuracy format cosine repetition 
--train_type lora  
--lora_rank 8  
--lora_alpha 32  
--vllm_device auto  
--vllm_gpu_memory_utilization 0.7 
--vllm_max_model_len 8192  
--torch_dtype bfloat16  
--dataset AI-MO/NuminaMath-TIR#5000  
--use_vllm true  
--max_completion_length 2048  
--num_train_epochs 1  
--per_device_train_batch_size 1  
--per_device_eval_batch_size 1  
--learning_rate 1e-6  
--gradient_accumulation_steps 2  
--eval_steps 200  
--save_steps 200  
--save_total_limit 2 
--logging_steps 5  
--max_length 4096  
--output_dir output  
--warmup_ratio 0.05  
--dataloader_num_workers 4  
--dataset_num_proc 4  
--num_generations 7  
--temperature 0.9  
--system 'examples/train/grpo/prompt.txt'  
--deepspeed zero2  
--tuner_backend unsloth

![Image](https://github.com/user-attachments/assets/0f54fddd-2793-449f-8b16-20ea8fe4f0ba)

GPU：A800 * 8

Can't unslot reduce the usage of gpu memory? Why is there still a oom?

**Comment by hjh0119** [comment]
unsloth is not support in grpo
